### PR TITLE
[MDE-2024] Adding new event files for DevopsDays Medellin 2024

### PR DESCRIPTION
**Pull request by mpinedag12**
[MDE-2024] Adding new event files for DevopsDays Medellin 2024

### CODE REVIEW

File: content/events/2024-medellin/welcome.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    </p>
+    </br>
+    <div class="d-flex flex-row">
+      <div class="col-md-12">
+        <div class="p-2">
+          <a
+            rel="noopener"
+            target="_blank"
+            class="btn btn-secondary btn-block"
+            href="https://latiquetera.com/evento/devopsdays-medellin-2023"
```
Comment (jyee) [inline content/events/2024-medellin/welcome.md:58]: This links to the 2023 registration

File: content/events/2024-medellin/welcome.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            style="background-color:#329cae; border-color:#72c1cd" onMouseOver="this.style.backgroundColor='#23738e'"
+            onMouseOut="this.style.backgroundColor='#329cae'"
+          >
+            <i class="fa fa-money fa-lg"></i>&nbsp;&nbsp;&nbsp;Sponsor the Conference
+          </a>
+        </div> -->
+        <div class="p-2">
+          <a
+            class="btn btn-secondary btn-block"
+            href="/events/2023-medellin/contact"
```
Comment (jyee) [inline content/events/2024-medellin/welcome.md:101]: This links to the 2023 organizer page.

File: data/events/2024-medellin.yml [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+# All dates are in unquoted YYYY-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow YYYY-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2024-04-27T08:00:00-06:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2024-04-28T08:00:00-06:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start:  2023-01-24T08:00:00-06:00 # start accepting talk proposals.
```
Comment (jyee) [inline data/events/2024-medellin.yml:17]: The CFP dates are all 2023. You should comment out the dates until you have correct ones for the 2024 event.

**Review by jyee — CHANGES_REQUESTED** [review]
Lots of old dates from copying files from previous years.

**Comment by FloorD** [comment]
@mpinedag12 can you look at the changes @jyee is suggesting?

File: content/events/2024-medellin/conduct.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
++++
+Title = "Conduct"
+Type = "event"
+Description = "Code of conduct for devopsdays Medellín 2022"
++++
+
+
+All attendees, speakers, sponsors, and volunteers at devopsdays Medellín 2022 are required to agree with the following code of conduct. Organizers will enforce this code throughout the event. We are expecting cooperation from all participants to help ensure a safe environment for everybody.
```
[the remark this answers is not in the document]
Comment (mpinedag12) [reply content/events/2024-medellin/conduct.md]: Date fixed.

File: content/events/2024-medellin/propose.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
++++
+Title = "Propose"
+Type = "event"
+Description = "Propose a talk for devopsdays Medellín 2022"
```
[the remark this answers is not in the document]
Comment (mpinedag12) [reply content/events/2024-medellin/propose.md]: Date fixed.

File: content/events/2024-medellin/speakers.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
++++
+Title = "Speakers"
+Type = "speakers"
+Description = "Speakers for devopsdays Medellin 2022"
```
[the remark this answers is not in the document]
Comment (mpinedag12) [reply content/events/2024-medellin/speakers.md]: Date fixed.

File: content/events/2024-medellin/welcome.md (hunk 1 above)
Comment (mpinedag12) [reply content/events/2024-medellin/welcome.md]: Date fixed.

File: data/events/2024-medellin.yml (hunk 3 above)
Comment (mpinedag12) [reply data/events/2024-medellin.yml]: Date fixed.

File: content/events/2024-medellin/conduct.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
++++
+Title = "Conduct"
+Type = "event"
+Description = "Code of conduct for devopsdays Medellín 2022"
```
[the remark this answers is not in the document]
Comment (mpinedag12) [reply content/events/2024-medellin/conduct.md]: Date fixed.

File: content/events/2024-medellin/registration.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
++++
+Title = "Registration"
+Type = "event"
+Description = "Registration for devopsdays Medellín 2021"
```
[the remark this answers is not in the document]
Comment (mpinedag12) [reply content/events/2024-medellin/registration.md]: Date fixed.

File: content/events/2024-medellin/sponsor.md [hunk 9]
```diff
@@ -0,0 +1,238 @@
++++
+Title = "Sponsor"
+Type = "event"
+Description = "Sponsor devopsdays Medellin 2021"
```
[the remark this answers is not in the document]
Comment (mpinedag12) [reply content/events/2024-medellin/sponsor.md]: Date fixed.

File: content/events/2024-medellin/welcome.md (hunk 2 above)
Comment (mpinedag12) [reply content/events/2024-medellin/welcome.md]: Date fixed.

File: content/events/2024-medellin/welcome.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+        <li style="margin-left:15px;">
+          <div>{{< event_start >}} - {{< event_end >}}</div>
+        </li>  
+      </ul>      
+    </p>
+    <h2>Conference format</h2>
+    <p>
+      <ul>
+        <li>
+          Three days of conferences and activities.
```
Comment (adrianmoisey) [inline content/events/2024-medellin/welcome.md:31]: I noticed that this says that the conference is 3 days, but only 2 dates are set.

Comment (mpinedag12) [reply content/events/2024-medellin/welcome.md]: Fixes and setting to Two Days

**Pull request merged**